libs/MailDB/include/MailDB/PgMailDB.h

```diff
@@ -17,32 +17,20 @@ class PgMailDB : public IMailDB
 
     virtual ~PgMailDB();
 
-    // TODO: Viacheslav
     virtual void Connect(const std::string &connection_string) override;
     virtual void Disconnect() override;
     virtual bool IsConnected() const override;
 
-    // TODO: Denys
-    virtual void SignUp(const std::string_view user_name, const std::string_view hash_password) override;
     virtual void Login(const std::string_view user_name, const std::string_view hash_password) override;
-    virtual std::string GetPasswordHash(const std::string_view user_name) override;
 
 
-    // TODO: Viacheslav
     virtual std::vector<User> RetrieveUserInfo(const std::string_view user_name = "") override;
```

Comment:
Use info.emplace_back instead of info.push_back;change return std::vector<User>(); to return {};